### DISCUSSION TITLE
Cannot Toggle SurfingKeys on current site - Arc Browser

### DISCUSSION THREAD

**Discussion by RyanfWood** [brookhong/Surfingkeys#2162] [General]
I have not been able to disable Surfingkeys for the last couple of days with the following shortcut:
Alt-s Toggle SurfingKeys on current site

Running in the Arc Browser
Version 1.60.0 (53678)
Chromium Engine Version 128.0.6613.138
macOS Sonoma 14.6.1 (23G93)

I also have Surfingkeys for Safari and it seems to be working correctly.
Anyone else having trouble with this? In the meantime I'm just hitting 'p' very frequently to bypass.

**Comment by brookhong** [comment]
Are you using Surfingkeys 1.17.0? If yes, please see https://github.com/brookhong/Surfingkeys/issues/2159

**Reply by RyanfWood** [reply]
Thank you. Worked perfectly. Sorry I didn't read those posts closely enough.
